lprint.c

```diff
@@ -341,7 +341,7 @@ printer_cb(const char     *device_info,	// I - Device information
       for (i = 2; i < 100; i ++)
       {
         snprintf(newname, sizeof(newname), "%s %d", name, i);
-        if (papplPrinterCreate(system, 0, name, driver_name, device_id, device_uri))
+        if (papplPrinterCreate(system, 0, newname, driver_name, device_id, device_uri))
```

Comment:
In reality, IPP limits the "printer-name" attribute to 127 octets (bytes), so the name/newname should be `char [128]`. In practice nobody uses names this long, but I should probably do something different here to safely come up with an alternate name that doesn't conflict instead of silently truncating...

Reply:
I see, and your fix also truncates correctly.
But I still don't see you using newname as I proposed? Am I missing something?

Reply:
Whoops, nope I just forgot to change the code I copied and pasted...

[master 8dcf23f] Fix typo with name collision fix.